libraries/AP_BoardConfig/AP_BoardConfig.cpp

```diff
@@ -337,6 +337,15 @@ const AP_Param::GroupInfo AP_BoardConfig::var_info[] = {
     AP_GROUPINFO("HEAT_LOWMGN", 23, AP_BoardConfig, heater.imu_arming_temperature_margin_low, HAL_IMU_TEMP_MARGIN_LOW_DEFAULT),
 #endif
 
+#if HAL_WITH_IO_MCU
```

Comment:
Should this just be a compile-time option?  Otherwise we have to carry both firmwares, don't we?

Reply:
Tend to agree. @bugobliterator what about doing this by default on the bdshot variant?

Reply:
We agreed to leave it this way for testing